CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+# Contributing to OpenCloud Docs
+
+Thank you for considering contributing to OpenCloud Docs! We appreciate your time and effort in helping improve our documentation.
+
+## How to Contribute
+
+### Reporting Bugs
+
+If you encounter a bug, please report it by [opening an issue](https://github.com/opencloud-eu/docs/issues/new). Include as much detail as possible to help us understand and address the issue promptly.
+
+### Suggesting Enhancements
+
+We welcome suggestions for improvements. To propose an enhancement, [open a new issue](https://github.com/opencloud-eu/docs/issues/new) and provide a clear and concise description of the enhancement and its benefits.
+
+### Submitting Pull Requests
+
+1. **Fork the Repository**: Click the "Fork" button at the top right of the repository page.
+
+2. **Clone Your Fork**:
+   ```bash
+   git clone https://github.com/opencloud-eu/docs.git
+   cd docs
+
+3. **Create a New Branch**:
+
+    ```bash
+    git checkout -b feature/your-feature-name
+
+4. **Make Your Changes**: 
+
+Implement your changes, adhering to the docs style.
+
+5. **Commit Your Changes**:
+
+    ```bash
+    git commit -m "Brief description of your changes"
+
+6. **Push to Your Fork**:
+
+    ```bash
+    git push origin feature/your-feature-name
+
+7. **Open a Pull Request**:
+
+Navigate to the original repository and click the "New Pull Request" button. Provide a detailed description of your changes.
+
+### Style Guides
+
+#### Branch names
+
+- Start the name with a capital letter ("Adjust Footer Links")
```

Comment:
@Svanvith That seems to be a misunderstanding. Branch names should be lowercase with no spaces.